keyboards/slimorta/keyboard.json

```diff
@@ -0,0 +1,89 @@
+{
+    "manufacturer": "ASHIJANKEN",
+    "keyboard_name": "Slimorta",
+    "maintainer": "ASHIJANKEN",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false
```

Comment:
```suggestion
        "mousekey": true
```

Reply:
Hello waffle87,
Thank you for reviewing my changes!
Could you tell me why I need to enable mousekey and remove nkro?

Reply:
`"nkro": false` does nothing as it's the default behaviour.
`"mousekey": true` allows users of QMK Configurator to use mousekeys.

Reply:
Thank you for your comment!
I don't use mousekey, but I understand it should be enabled for users of QMK Configurator.